specification/network/resource-manager/Microsoft.Network/stable/2022-01-01/network.json

```diff
@@ -168,6 +168,33 @@
       "description": "Common resource representation.",
       "x-ms-azure-resource": true
     },
+    "ProxyResource": {
```

Comment:
nit: is 'Proxy' really the right word? 'Child' seems more appropriate https://armwiki.azurewebsites.net/introduction/concepts/resources.html#resource-terminology